widget/numerical_entry.go

```diff
@@ -72,12 +177,253 @@ func (e *NumericalEntry) Keyboard() mobile.KeyboardType {
 	return mobile.NumberKeyboard
 }
 
-func (e *NumericalEntry) isNumber(content string) bool {
-	if e.AllowFloat {
-		_, err := strconv.ParseFloat(content, 64)
-		return err == nil
+// Unbind disconnects the entry from the bound data.
+// This will remove the data updates and allow the entry to be used independently.
+func (e *NumericalEntry) Unbind() {
+	e.binder.Unbind()
+}
+
+// ValidateText checks if the entered text is a valid numerical input
+// according to the system locale.
+func (e *NumericalEntry) ValidateText(text string) error {
```

Comment:
It doesn't feel like this should be public API.

Reply:
Changed to validateText.
Validator set to validateText in appropriate places if Validator is nil.